pysat/_meta.py

```diff
@@ -467,6 +467,7 @@ def __setitem__(self, data_vars, input_dat):
                     # metadata. Meta inputs could be part of a larger multiple
                     # parameter assignment, so not all names may actually have
                     # a 'meta' object to add.
+                    self.warn_meta_children()
```

Comment:
In the other PR (#970), @rstoneback suggested we remove this since it will always raise the warning at line 493 in a recursive fashion.  Running the tests individually confirms that 493 is always raised.

Reply:
dropping 470